### ISSUE TITLE
WCAG 1.3.1,WCAG 4.1.2: Ensures select element has an accessible name (select)

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode-pull-request-github#4192]
Tags: Accessibility, WCAG 1.3.1, WCAG 4.1.2, select-name

Issue: Ensures select element has an accessible name (select-name - https://dequeuniversity.com/rules/axe/4.4/select-name?application=msftAI)

Element path: #webview-419b4e8e-bb4d-433b-857b-78d8109235f9 > .webview.ready;#active-frame;select

Snippet: `<select><option value="merge">Create Merge Commit</option><option value="squash" selected="">Squash and Merge</option><option value="rebase">Rebase and Merge</option></select>`

How to fix: 
Fix any of the following:
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

![image](https://user-images.githubusercontent.com/38270282/201961764-4369bcbf-cfdc-4ff9-98f6-41f25176e5ce.png)
